py-polars/src/series.rs

```diff
@@ -1123,6 +1123,11 @@ impl PySeries {
         Ok(n)
     }
 
+    pub fn approx_unique(&self) -> PyResult<Self> {
```

Comment:
This should be removed. (expression API)

Reply:
done